sbt-dotty/sbt-test/sbt-dotty/example-project/test

```diff
@@ -1,4 +1,3 @@
 > run
 > 'set initialCommands := "1 + 1" '
-# FIXME: does not work on the CI
-#> console
+> console
```

Comment:
A more extensive REPL test would be nice. I don't know to what extent we can test the REPL with scripted tests. E.g. can we test something like #3007?

Reply:
IIUC, this already tests that starting the console succeeds, which seems why @SzymonPajzert was interested for #4933.

If we want to connect something to the actual terminal, give commands and wait for responses, one could use `expect`-like programs.
If we bypass the terminal logic, it seems we'd have to add a `consoleTest script` command which combines `console` with `ReplTest`: it'd be a `ConsoleInterface` creating, instead of `ReplDriver`, a `ScriptedTests` on the script.